libc/src/math/generic/CMakeLists.txt

```diff
@@ -111,7 +111,7 @@ add_entrypoint_object(
   DEPENDS
     libc.src.__support.macros.properties.types
     libc.src.__support.FPUtil.nearest_integer_operations
-    libc.src.__support.macros.properties.architectures
+    libc.src.__support.macros.properties.cpu_features
```

Comment:
Do you mind running the bazel builds to see if these extra dependencies need to be added to bazel overlay?

Reply:
Bazel build seems to work:

```
~/projects/llvm-project/utils/bazel libc-math-float16-gpu ➜ bazel build --config=generic_clang @llvm-project//libc/...
INFO: Analyzed 897 targets (68 packages loaded, 6253 targets configured).
INFO: Found 897 targets...
INFO: Elapsed time: 166.512s, Critical Path: 151.57s
INFO: 3876 processes: 2132 internal, 1744 linux-sandbox.
INFO: Build completed successfully, 3876 total actions
~/projects/llvm-project/utils/bazel libc-math-float16-gpu ➜ bazel test --config=generic_clang @llvm-project//libc/...
INFO: Analyzed 897 targets (0 packages loaded, 0 targets configured).
INFO: Found 659 targets and 238 test targets...
INFO: Elapsed time: 5.366s, Critical Path: 4.26s
INFO: 239 processes: 476 linux-sandbox.
INFO: Build completed successfully, 239 total actions
@llvm-project//libc/test/src/__support:arg_list_test                     PASSED in 0.0s
...

Executed 238 out of 238 tests: 238 tests pass.
There were tests whose specified size is too big. Use the --test_verbose_timeout_warnings command line option to see which ones these are.
~/projects/llvm-project/utils/bazel libc-math-float16-gpu ➜
```

Why are only some math functions listed in https://github.com/llvm/llvm-project/blob/main/utils/bazel/llvm-project-overlay/libc/BUILD.bazel?

Reply:
B/c I've been lazy / forget to update it, especially since there are quite some work need to be done to add smoke tests to the lists.  I planned to update it all at once at some point in late July, early August.